lib/engine/game/g_18_usa/game.rb

```diff
@@ -654,24 +712,29 @@ def next_round!
               reorder_players
               new_operating_round
             when Engine::Round::Operating
+              # The normal export logic which plays nicely with privates exists in first_turn_housekeeping but it does not
+              #  work in the edgecase where there are 0 entities acting in an operating round - this exists to cover this
+              #  edge case. Since there are no corporatiosn, there is nobody who could use a private to save a train from
+              #  rusting prematurely so this is fine.
+              export_train if @round.entities.empty?
```

Comment:
I fixed this too. Its already in master.

My fix was to move the logic to @round.finished? instead of @round.after_process.

Reply:
You can remove this one.

Reply:
I've rebased the branch against master and removed this